src/videojs-http-streaming.js

```diff
@@ -189,6 +191,42 @@ const setupEmeOptions = (hlsHandler) => {
   }
 };
 
+const getVhsLocalStorage = () => {
+  if (!window.localStorage) {
+    return null;
+  }
+
+  const storedObject = window.localStorage.getItem('videojs-vhs');
+
+  if (!storedObject) {
+    return null;
+  }
+
+  return JSON.parse(storedObject);
```

Comment:
this probably needs to be try/catched as well as it's possible someone tampered with the value.